Assets/RedBlueGames/BulkRename/Editor/BulkRenamerWindow.cs

```diff
@@ -211,64 +210,33 @@ private void OnGUI()
             EditorGUILayout.EndScrollView();
         }
 
-        private string[] GetNamesFromSelections()
+        private void SortObjectsToRename()
         {
-
-            List<string> allAssetNames = new List<string>();
-            List<Object> allSortedGameObectsNames = new List<Object>();
-
-            int namesCount = this.objectsToRename.Count;
-            var names = new string[namesCount];
-            bool isItAsset = AssetDatabase.GetAssetPath(objectsToRename[0]) != "";
```

Comment:
![problem with names](https://cloud.githubusercontent.com/assets/20247639/24199406/b2efb920-0f09-11e7-9830-d10c010e4379.png)
I don't have names of assets. That's why I don't use Object.name.